### PR TITLE
Chore/Migrate to vite

### PR DESCRIPTION
**Pull request by igor-dmscn**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,11 @@ package-lock.json
 .nvmrc
 .rspec
 .tool-versions
+
```
Comment (talyssonoc) [inline .gitignore:34]: We probably can remove the folders `public/packs`, `public/packs-test` and `public/webpack` as well from this file, right?

File: spec/feature_helper.rb [hunk 2]
```diff
@@ -1,11 +1,6 @@
 require 'rails_helper'
 
 RSpec.configure do |config|
-  config.before(:suite) do
-    `bundle exec rake assets:precompile`
-    Webpacker.compile
-  end
```
Comment (talyssonoc) [inline spec/feature_helper.rb:7]: Don't we need any alternative to that with Vite?

File: Procfile.dev [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
```
[the remark this answers is not in the document]
Comment (igor-dmscn) [reply Procfile.dev:1]: removed

File: .gitignore (hunk 1 above)
Comment (igor-dmscn) [reply .gitignore:34]: removed the ones I found

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -11,6 +11,18 @@
...
+    <!--
+      If using a TypeScript entrypoint file:
+        vite_typescript_tag 'application'
+
+      If using a .jsx or .tsx entrypoint, add the extension:
+        vite_javascript_tag 'application.jsx'
+
+      Visit the guide for more information: https://vite-ruby.netlify.app/guide/rails
+    -->
+
```
[the remark this answers is not in the document]
Comment (igor-dmscn) [reply app/views/layouts/application.html.erb:25]: done

File: package.json [hunk 5]
```diff
@@ -111,17 +110,14 @@
     "reselect": "^4.0.0",
     "resolve-url-loader": "^5.0.0",
```
[the remark this answers is not in the document]
Comment (igor-dmscn) [reply package.json:111]: removed some, I'm not confident enough to remove all of them lol

File: app/helpers/application_helper.rb [hunk 6]
```diff
@@ -7,4 +7,27 @@ def team_logo(image_path)
       radius: 5,
       border: '1px_solid_black')
   end
+
+  CLOUDINARY_JS_CONFIG_PARAMS = %i[
+    api_key,
+    cloud_name,
+    private_cdn,
+    secure_distribution,
```
Comment (talyssonoc) [inline app/helpers/application_helper.rb:15]: Found this bug while testing in staging, when using this syntax we should not include the comma:

```suggestion
    api_key
    cloud_name
    private_cdn
    secure_distributio
```

**Pull request merged**